app/workers/rss_reader_fetch_user_worker.rb

```diff
@@ -0,0 +1,11 @@
+class RssReaderFetchUserWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :medium_priority
```

Comment:
Good call not limiting the retries since this is making external requests to another service